spec/integration/models/project_spec.rb

```diff
@@ -204,7 +204,7 @@
   end
 
   describe '#import_clone' do
-    let(:clone) { GoodData::Project.create(title: 'import clone test', client: @client, auth_token: ConnectionHelper::GD_PROJECT_TOKEN) }
+    let(:clone) { GoodData::Project.create(title: 'import clone test', client: @client, auth_token: ConnectionHelper::SECRETS[:gd_project_token]) }
```

Comment:
warning: Similar code found in project_spec.rb:166, project_spec.rb:207 (mass = 32) (flay)